server/odc-service/src/main/java/com/oceanbase/odc/service/db/DBTablePartitionService.java

```diff
@@ -40,7 +41,8 @@ public List<DBTablePartitionResponse> list(@NonNull ConnectionSession connection
         String schemaName = ConnectionSessionUtil.getCurrentSchema(connectionSession);
         DBTablePartition partition = accessor.getPartition(schemaName, tableName);
         return partition.getPartitionDefinitions().stream()
-                .map(DBTablePartitionResponse::new).collect(Collectors.toList());
+                .map(DBTablePartitionResponse::new)
```

Comment:
partition can not be ordered by its name

Reply:
ok，remove it